test/storage/delta_record_test.cpp

```diff
@@ -0,0 +1,126 @@
+#include <unordered_map>
+#include <utility>
+#include <vector>
+#include "common/object_pool.h"
+#include "storage/data_table.h"
+#include "storage/storage_util.h"
+#include "util/storage_test_util.h"
+#include "storage/storage_defs.h"
+#include "util/test_harness.h"
+
+namespace terrier {
+
+struct DeltaRecordTests : public ::terrier::TerrierTest {
```

Comment:
nit: you are already under terrier namespace so `struct DeltaRecordTests : public TerrierTest` would suffice